app/messaging/email/create-submission-msg.js

```diff
@@ -198,19 +199,44 @@ function scoreQuestions(submission, desirabilityScore) {
     collaborationScore: getQuestionScoreBand(desirabilityScore.desirability.questions, 'collaboration'),
     environmentalImpact: [submission.environmentalImpact].flat().join(', '),
     environmentalImpactScore: getQuestionScoreBand(desirabilityScore.desirability.questions, 'environmental-impact'),
+    mechanisation: submission.mechanisation,
+    mechanisationScore: getQuestionScoreBand(desirabilityScore.desirability.questions, 'mechanisation'),
```

Comment:
mechanisation doesn't have a score. Use **manualLabour** if it is there, else say "Weak"